tests/Console/ModelsCommand/GeneratePhpdocWithFqnInExternalFile/Builders/EMaterialQueryBuilder.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
May I suggest we (this package) always tries to use `declare(strict_types=1);`, at least in all the tests? thanks!

Reply:
Let's add a phpcs rule for this then! https://github.com/slevomat/coding-standard#slevomatcodingstandardtypehintsdeclarestricttypes-

We can configure it just for the tests dir

Reply:
Declare added

Reply:
@mr-feek 🤗